### PR TITLE
Detail links

### PR DESCRIPTION
**Pull request by zack-ashen**
### Summary 

Make it possible to visit student detail and rider detail pages directly via link. 

### Test Plan 

Difficult to demo this feature...

### CODE REVIEW

File: frontend/src/components/EmployeeCards/EmployeeCards.tsx [hunk 1]
```diff
@@ -79,7 +57,7 @@ const EmployeeCard = ({
 
 
   return (
-    <Link to={{ pathname: '/employees/employee', state: userInfo, search: `?name=${fullName}` }}
+    <Link to={{ pathname: '/employees/employee/' + userInfo.id, state: userInfo }}
```
Comment (mg2239) [inline frontend/src/components/EmployeeCards/EmployeeCards.tsx:60]: you can make the pathname /admins or /drivers depending on the user, that way you wont have to make two requests later on in EmployeeDetail. just be sure to change the routes accordingly too

File: frontend/src/components/UserTables/StudentsTable.tsx [hunk 2]
```diff
@@ -37,16 +37,16 @@ const StudentsTable = () => {
...
           lastName,
           netID: netId,
           phone,
           accessibility: disability,
         };
         const location = {
-          pathname: '/riders/rider',
-          state: riderData,
-          search: `?name=${`${firstName}_${lastName}`}`,
+          pathname: '/riders/rider/' + r.id,
```
Comment (mg2239) [inline frontend/src/components/UserTables/StudentsTable.tsx:48]: this can also just be /riders/ + r.id

**Review by mg2239 — CHANGES_REQUESTED** [review]
works great, just a couple small changes 👍

**Review by zack-ashen — COMMENTED** [review]
I think I made the changes you requested...

**Review by mg2239 — APPROVED** [review]
looks good, i did the route changes i was suggesting (basically making the route /admin/admins, /admin/drivers, or /admin/riders so you can easily identify the user type) and changed the routing accordingly

**Pull request merged**